docs/admin/configuration/authentication-and-user-management/index.md

```diff
@@ -7,15 +7,41 @@ title: 'Authentication and Identity Management'
 OpenCloud employs a dual-track authentication strategy:
 
 1. **Built-in IDP (Identity Provider)**:
-   - Based on LibreGraph Connect (lico)
+   - Based on LibreGraph Connect (Lico)
    - Targeted at smaller installations (up to 500 users)
    - Designed for standalone or small deployments
 
-2. **External Identity Providers**:
+2. **External IDP**:
    - Keycloak as the recommended OIDC provider for larger installations
    - Support for Azure AD, EntraID, ADFS through Keycloak
    - Enterprise-focused solution
 
+## Your Use Cases
+
+### Choose Lico if you need:
+- Simple OpenID Connect provider
+- Lightweight, fast solution
+- Small to medium deployments
+- Integration with ownCloud/LibreGraph ecosystem
```

Comment:
This is rather unspecific. What is the ownCloud/LibreGraph ecosystem, and why ownCloud? I guess you could just leave it out.